pyiron/atomistics/structure/atoms.py

```diff
@@ -1058,115 +1058,208 @@ def analyse_phonopy_equivalent_atoms(atoms):
+               select_atoms=None, background='white', color_scheme=None,
+               scalar_field=None, vector_field=None, vector_color=None,
+               custom_array=None, custom_3darray=None):
         """
+        Plot3d relies on NGLView to visualize atomic structures. Here, we construct a string in the "protein database"
+        ("pdb") format, then turn it into an NGLView "structure". PDB is a white-space sensitive format, so the
+        string snippets are carefully formatted.
+
+        The final widget is returned. If it is assigned to a variable, the visualization is suppressed until that
+        variable is evaluated, and in the meantime more NGL operations can be applied to it to modify the visualization.
 
         Args:
-            spacefill:
-            show_cell: whether to show the frame or not (default: True)
-            camera: 'perspective' or 'orthographic'
-            particle_size: particle size
-            background: back ground color
-            color_scheme: v.i.
-            scalar_field: color for each atom according to the array value
-            vector_field: vectors for each atom according to the array values (3 values for each atom)
-            vector_color: vector for color coding (only available with vector_field)
-            select_atoms: atoms to show (1d array with ID's or True for atoms to show)
-
-            Possible color schemes: 
+            show_cell (bool): Whether or not to show the frame. (Default is True.)
+            show_axes (bool): Whether or not to show xyz axes. (Default is True.)
+            camera (str): 'perspective' or 'orthographic'. (Default is 'perspective'.)
+            spacefill (bool): Whether to use a space-filling or ball-and-stick representation. (Default is True, use space-filling atoms.)
+            particle_size (float): Size of the particles. (Default is 1.)
+            select_atoms (ndarray): Indices of atoms to show, either as integers or a boolean array mask. (Default is None, show all atoms)
+            background (str): Background color. (Default is 'white'.)
+            color_scheme (str): NGLView color scheme to use. (Default is None,
+            scalar_field (ndarray): Color each atom according to the array value (Default is None, use coloring scheme.)
+            vector_field (ndarray): Add vectors (3 values) originating at each atom. (Default is None, no vectors.)
+            vector_color (ndarray): Colors for the vectors (only available with vector_field). (Default is None, vectors are colored by their direction.)
+
+            Possible NGLView color schemes:
               " ", "picking", "random", "uniform", "atomindex", "residueindex",
               "chainindex", "modelindex", "sstruc", "element", "resname", "bfactor",
               "hydrophobicity", "value", "volume", "occupancy"
     
         Returns:
-    
+            (NGLWidget): The NGLView widget itself, which can be operated on further or viewed as-is.
```

Comment:
`nglview.NGLWidget` for proper docstring indexing

Reply:
Btw, thank you so much for the detailed and clear docstrings for the`plot3d()`! :)

Reply:
Also done.

np. Glad to hear they're sufficiently clear. One can be detailed, but sometimes more detail doesn't help clarity ;)